tools/c7n_azure/c7n_azure/resources/key_vault.py

```diff
@@ -114,3 +119,48 @@ def enhance_policies(self, access_policies):
             policy['principalName'] = GraphHelper.get_principal_name(aad_object)
 
         return access_policies
+
+
+@resources.register('key')
+class KeyVaultKey(ArmResourceManager):
+
+    class resource_type(ArmResourceManager.resource_type):
+        service = 'azure.mgmt.keyvault'
+        client = 'KeyVaultManagementClient'
+        enum_spec = ('vaults', 'list', None)
+
+    def augment(self, resources):
+        url_base = 'vault.azure.net'
```

Comment:
Please move `https://vault.azure.net` to the constants file.

Reply:
Done, will be in next commit.